metrics/package.go

```diff
@@ -140,9 +140,11 @@ func parseDpkgOutput(packageName, dpkgOutput string, dpkgErr error) (*Package, e
 
 		tokens := strings.Split(line, " ")
 		// The successful line for package shall be in format:
-		// <package name> <status> <version>.
+		// <package name> <status> [epoch:]<version>.
 		// Example:
 		// 'percona-xtrabackup-81 ii 8.1.0-1-1.jammy'
+		// or with epoch:
+		// 'percona-xtrabackup-81 ii 2:8.1.0-1-1.jammy'
```

Comment:
I wonder if it's some "well known" format and there are some existing parsers?

Reply:
format well-known https://www.debian.org/doc/debian-policy/ch-controlfields.html#version, parsers exist. the problem - we do not follow this format in our packages :(